scene/gui/reference_rect.h

```diff
@@ -37,6 +37,7 @@ class ReferenceRect : public Control {
 
 	GDCLASS(ReferenceRect, Control);
 	Color border_color;
+	bool show_in_game = false;
```

Comment:
Its looks like C++11 initialization (Godot still uses 03), instead 'show_in_game = false' should be placed within class constructor in cpp file.